pages/portfolio.html

```diff
@@ -4,9 +4,82 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link href="../styles/styles.css" rel="stylesheet">
+  <link href="../styles/portfolio.css" rel="stylesheet">
+  <title>JFD Ada Personal Portfolio portfolio</title>
 </head>
-<body>
-  
+<body class="container">
+  <header>
+    <h1>Joanna Dudley</h1>
+    <h4 class="header-subtitle">portfolio</h4>
+  </header>
+  <nav>
+    <ul>
+      <li><a href="index.html">Homepage</a></li> 
+      <li><a href="about.html">About Me</a></li> 
+      <li><a href="#.html">Portfolio</a></li> 
+    </ul>
+  </nav>
+  <main class="nested-container">
+    <section class="description left-column">
+      <h3>Task List API</h3>
+      <p>A web API created to organize tasks and goals, with an integrated slackbot. <a href="https://ada-jfd-task-list-api.herokuapp.com/tasks">Tasks</a> can be sorted, filtered, marked complete, and added to <a href="https://ada-jfd-task-list-api.herokuapp.com/goals">goals</a>. Project deployed on Heroku.</p>
+      <h4>Skills Assessed</h4>
+      <ul>
+        <li>Demonstrating understanding of the client-server model, request-response cycle and conventional RESTful routes</li>
+        <li>Reading and using existing external web APIs</li>
+        <li>Creating databases and models</li>
+        <li>Create conventional RESTful routes as well as unconventional routes for custom behavior</li>
+        <li>Creating a one-to-many relationship between two models</li>
+        <li>Writing tests</li>
+        <li>etc</li>
+      </ul>
+    </section>
+    <div class="git-link right-column"><a href="https://github.com/jfdudley/task-list-api"><img src="../images/noun-task-list-2154088.png" alt="checklist icon"></a></div> 
+    <div class="git-link left-column"><a href="https://github.com/jfdudley/solar-system-api"><img src="../images/noun-solar-system-3328563.png" alt="solar system icon"></a></div> 
```

Comment:
This is another place where I might see an argument for `div`, but to me, it's really grouping together a section of a project row that contains the related project image and link, so I would suggest using `section` over a `div` here.

Reply:
These were also flagged by the html checker when I had them as sections, since there's no text at all, so I changed them.